zreport.go

```diff
@@ -14,4 +34,140 @@ type ZReportInfo struct {
 	LastReceiptSeq   int         `json:"LastReceiptSeq"`
 }
 
-/// here methods to implement zReport endpoints
\ No newline at end of file
+type dateTime struct {
+	DateTime string `json:"DateTime"`
+}
+
+// OpenZreport opens a new Zreport for the fiscal drive
+// createdTime is in format "2006-01-02 15:04:05" or "now"
+func (o *ofdConnector) OpenZreport(ctx context.Context, createdTime string) error {
+	if o.factoryID == "" {
```

Comment:
It looks like factoryID should be set no matter what. Could we probably move this check to ofdConnector/Service constructor?

Reply:
I bet if we won't set this variable we can't work. If this variable is passed with every call we should add it to method call. if it is set from client code of ofd_sample - we should move it to constructor i guess